crates/node/src/interactive_cli/webui.rs

```diff
@@ -15,20 +13,14 @@ impl WebUICommand {
         let addr = node
             .server_config
             .listen
-            .get(0)
+            .first()
             .ok_or_else(|| eyre!("No listen address found"))?;
```

Comment:
Since `server_config.listen` is a list of multiaddrs, you can loop through them all, converting each one until we find one that is valid

something like this

```rust
let mut urls = node.server_config.listen
    .iter()
    .map(|addr| multiaddr_to_url(addr, "/admin-dashboard"))
    .peekable();

let addr = 'done: {
    while let Some(url) = urls.next() {
        match addr {
            Ok(url) => break 'done url,
            Err(err) if urls.peek().is_none() => bail!(err),
            Err(_) => continue,
        };
    }

    bail!("no listen address found");
};

webbrowser::open(url.as_str()).wrap_err("Failed to open browser")?;

// ..
```

essentially making sure we check all multiaddrs before failing